README.md

```diff
@@ -1,38 +1,38 @@
-<p align="center">
-  <a href="http://getmobicss.com/">
-    <img height="200" alt="Mobi.css Logo" src="http://getmobicss.com/img/mobi-logo.png"/>
-  </a>
-</p>
-
-# [Mobi.css](http://getmobicss.com)
-
-[![Build Status](https://img.shields.io/travis/mobi-css/mobi.css.svg)](https://travis-ci.org/mobi-css/mobi.css) [![npm package](https://img.shields.io/npm/v/mobi.css.svg)](https://www.npmjs.org/package/mobi.css) [![npm downloads](http://img.shields.io/npm/dm/mobi.css.svg)](https://www.npmjs.org/package/mobi.css)
-
-> A lightweight, scalable, mobile-first css framework
-
-## Features
-
-### Lightweight
-
-The default bundle of Mobi.css provides the basic styles to build clean and elegant webpages. It's **only 3kb** after gzipped, and it's very easy to get started.
-
-### Scalable
-
-Plugin system makes css scalable. You can import plugins to enhance Mobi.css, or use the plugin as a standalone css library. **It's even possible to use a plugin without Mobi.css**.
-
-### Mobile-first
-
-Mobi.css focus on the details and provides the best user experience, especially for content-rich webpage. It's designed for mobile, but also works awesome on desktop.
-
-## Community
-
-Like Mobi.css? Please join the community to discuss new features, open an issue or create pull requests!
-
-- [GitHub organization](http://github.com/mobi-css)
-- [Follow us on twitter](https://twitter.com/mobi_css)
-- [Join gitter discuss](https://gitter.im/mobi-css)
-- [Read our blogs](https://medium.com/@mobi_css)
-
-## License
-
-[MIT](https://github.com/mobi-css/mobi.css/blob/master/LICENSE)
+<p align="center">
+  <a href="http://getmobicss.com/">
+    <img height="200" alt="Mobi.css Logo" src="http://getmobicss.com/img/mobi-logo.png"/>
+  </a>
+</p>
+
+# [Mobi.css](http://getmobicss.com)
+
+[![Build Status](https://img.shields.io/travis/mobi-css/mobi.css.svg)](https://travis-ci.org/mobi-css/mobi.css) [![npm package](https://img.shields.io/npm/v/mobi.css.svg)](https://www.npmjs.org/package/mobi.css) [![npm downloads](http://img.shields.io/npm/dm/mobi.css.svg)](https://www.npmjs.org/package/mobi.css)
+
+> A lightweight, scalable, mobile-first css framework
+
+## Features
+
+### Lightweight
+
+The default bundle of Mobi.css provides the basic styles to build clean and elegant webpages. It's **only 3kb** after gzipped, and it's very easy to get started.
+
+### Scalable
+
+Plugin system makes css scalable. You can import plugins to enhance Mobi.css, or use the plugin as a standalone css library. **It's even possible to use a plugin without Mobi.css**.

```

Comment:
really minor => css need to be capital letter in readme: CSS.
(I checked Bootstrap, Foundation... they all use capital letter CSS 😄 )

Reply:
Fixed!